app/models/like.rb

```diff
@@ -0,0 +1,14 @@
+class Like < ApplicationRecord
+  belongs_to :user, foreign_key: 'author_id'
+  belongs_to :post
+
+  validates :author_id, presence: true
+  validates :post_id, presence: true
+
+  after_save :update_likes
+
+  def update_likes
+    post = self.post
+    post.update(likes_counter: post.likes.count)
+  end
```

Comment:
- [ OPTIONAL] I recommend keeping the update likes method private since you used one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - `after_save` to call the `#update_likes_counter` method every time a new like has been saved to database, so unlike the previous reviewer I think it would be better if you keep as you make It first 💪🏻✅👏🏻💯🔥

Reply:
noted